Sources/SwiftLayout/Layoutable/Wrapper/SL.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  SL.swift
+//  
+//
+//  Created by oozoofrog on 2022/03/13.
+//
+
+import Foundation
+
+public final class SL<L: Layoutable> {
+    private weak var layoutable: L?
+    
+    init(_ layoutable: L) {
+        self.layoutable = layoutable
+    }
+    
+    public func layout() {
```

Comment:
In fact, also the name "layout" is ambiguous.
Even if sl is attached, a clear name like "updateLayout" would be better.

Reply:
agreed